kv/index_test.go

```diff
@@ -33,6 +38,102 @@ func Test_Bolt_Index(t *testing.T) {
 	influxdbtesting.TestIndex(t, s)
 }
 
+func TestIndex_Walk(t *testing.T) {
+	t.Run("only selects exact keys", func(t *testing.T) {
+		ctrl := gomock.NewController(t)
+		t.Cleanup(ctrl.Finish)
```

Comment:
take it or leave it: With 1.14+ this happens by default:
https://godoc.org/github.com/golang/mock/gomock#NewController
```suggestion
```

Reply:
I have been waiting for this, however, the docs are misleading as there is no released version of `gomock` with this support yet:

mock@master has the `Cleanup` support:

https://github.com/golang/mock/blob/99aa9272d551d0d16f4bdcc43dc06a3077e69b3a/gomock/controller.go#L136-L141

latest tagged version is mock@1.4.4, which does not have `Cleanup` support:

https://github.com/golang/mock/blob/f7b1909c82a8958747e5c87c6a5c3b2eaed8a33d/gomock/controller.go#L118-L128